lib/gym.rb

```diff
@@ -35,7 +30,34 @@ def gymfile_name
     def xcode_path
       @path ||= `xcode-select --print-path`.strip
     end
+
+    def xcode_version
+      @version ||= parse_version
+    end
+
+    def pre_7?
+      v = xcode_version
+      is_pre = v.split('.')[0].to_i < 7
+      is_pre
+    end
+
+    def init_libs
+      return unless pre_7?
+      # Import all the fixes
+      require 'gym/xcodebuild_fixes/swift_fix'
+      require 'gym/xcodebuild_fixes/watchkit_fix'
+      require 'gym/xcodebuild_fixes/package_application_fix'
+    end
+
+    private
+
+    def parse_version
+      output = `DEVELOPER_DIR='' "#{xcode_path}/usr/bin/xcodebuild" -version`
+      return '0.0' if output.nil?
```

Comment:
I would probably go for error. I like a fail fast approach. Not being able to parse the xcodebuild output is rather serious and I am not sure I want to make the current caller more complex
`````
def xcode_version
  @version ||= parse_version
end
`````